package.json

```diff
@@ -176,6 +176,7 @@
     "pnpm": "^8.6.6",
     "prettier": "^2.8.8",
     "rollup-plugin-sourcemaps": "^0.6.3",
+    "stream-browserify": "^3.0.0",
```

Comment:
We have a vite fallback but for some reason weren't requiring this so dev/build were failing